consensus/replay_test.go

```diff
@@ -1062,13 +1063,23 @@ func newMockBlockStore(config *cfg.Config, params types.ConsensusParams) *mockBl
 
 func (bs *mockBlockStore) Height() int64                       { return int64(len(bs.chain)) }
 func (bs *mockBlockStore) LoadBlock(height int64) *types.Block { return bs.chain[height-1] }
+func (bs *mockBlockStore) LoadBlockByHash(hash libBytes.HexBytes) *types.Block {
```

Comment:
same question here

Reply:
If I will change it to [ ]bytes, which I already did earlier to create PR this gives me an error that, the type is not correct. Maybe I will change type wherever necessary to make this happen.